File: TrainingAndroidWithJava/app/src/main/java/com/example/trainingandroidwithjava/models/UserModel.java
package com.example.trainingandroidwithjava.models;

public final class user {
    private String name = "";
    private String email = "";
    private int cellphone = 0;

    public user() {}

    public user(String name, String email, int cellphone) {
        this.name = name;
        this.email = email;
        this.cellphone = cellphone;
    }

    public String getName(){
        return this.name;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getEmail(){
        return this.email;
    }

    public void setEmail(String email){
        this.email = email;
    }

    public int getCellphone(){
        return this.cellphone;
    }

    public void setCellphone(int cellphone){
        this.cellphone = cellphone;
    }
}

File: ReviewForTest/app/src/main/java/com/example/reviewfortest/models/userModel.kt
package com.example.reviewfortest.models

class userModel {
    private var cellphone : String = ""
    private var email : String = ""
    private var name : String = ""
    private var password : String = ""

    fun userModel () {}

    fun getCellphone () : String {
        return this.cellphone
    }

    fun getEmail () : String {
        return this.email
    }

    fun getName () : String {
        return this.name
    }

    fun getPassword () : String {
        return this.password
    }

    fun setCellphone (cellphone : String) {
        this.cellphone = cellphone
    }

    fun setEmail (email : String) {
        this.email = email
    }

    fun setName (name : String) {
        this.name = name
    }

    fun setPassword (password : String) {
        this.password = password
    }

    override fun toString () : String {
        return "cellphone: ${this.cellphone},\n email: ${this.email},\n name: ${this.name},\n password: ${this.password} \n"
    }
}
